src/main/java/com/kjetland/dropwizard/activemq/ActiveMQReceiver.java

```diff
@@ -2,5 +2,5 @@
 
 public interface ActiveMQReceiver<T> {
 
-    public void receive(T message);
+    void receive(T message, String identifier);

```

Comment:
If you take into account my comments in the service-utils PR, the identifier parameter would have to be replaced by a map of all the message properties. Apparently there is a `message.getPropertyNames()` method available. 😉
Maybe we  could also have a parameter object as the argument: message + properties.

Reply:
I very much agree, at first it wasn't obvious where to "hide" what functionality but I think your suggestion is better. I will move some things around :)